# 마지막으로 인덱스 오류를 살펴보겠습니다.

a = [1,2,3]
print(a[4])

# 이번에는 a라는 리스트에 없는 리스트 값을 불러왔더니 'IndexError'명의 에러가 발생했다.

# 이 예외 처리 단원에서 배울 것은 오류 예외 처리 기법입니다.